addons/claim_from_delivery/models/stock_picking.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    claim_count_out = fields.Integer(compute='_compute_claim_count_out', string='Claims')
+
+    @api.multi
+    def _compute_claim_count_out(self):
+        for picking in self:
+            data = self.env['crm.claim'].read_group([('ref', '=', ('stock.picking,' + str(picking.id)))], ['ref'], ['ref'])
```

Comment:
What a bad idea... a read_group in a loop...

Read_group is useful when you have more than one object to group on. Here, keeping the search_count would have been ok